[2025-03-01T11:09:48.111+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: customer_churn_ml_pipeline.data_storage_task manual__2025-03-01T11:08:35.287819+00:00 [queued]>
[2025-03-01T11:09:48.122+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: customer_churn_ml_pipeline.data_storage_task manual__2025-03-01T11:08:35.287819+00:00 [queued]>
[2025-03-01T11:09:48.123+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-03-01T11:09:48.138+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): data_storage_task> on 2025-03-01 11:08:35.287819+00:00
[2025-03-01T11:09:48.143+0000] {standard_task_runner.py:57} INFO - Started process 13014 to run task
[2025-03-01T11:09:48.146+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'customer_churn_ml_pipeline', 'data_storage_task', 'manual__2025-03-01T11:08:35.287819+00:00', '--job-id', '117', '--raw', '--subdir', 'DAGS_FOLDER/customer_churn.py', '--cfg-path', '/tmp/tmp8p_5psqy']
[2025-03-01T11:09:48.148+0000] {standard_task_runner.py:85} INFO - Job 117: Subtask data_storage_task
[2025-03-01T11:09:48.169+0000] {logging_mixin.py:154} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-01T11:09:48.213+0000] {task_command.py:416} INFO - Running <TaskInstance: customer_churn_ml_pipeline.data_storage_task manual__2025-03-01T11:08:35.287819+00:00 [running]> on host 7e994a0ed489
[2025-03-01T11:09:48.322+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='customer_churn_ml_pipeline' AIRFLOW_CTX_TASK_ID='data_storage_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-01T11:08:35.287819+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-01T11:08:35.287819+00:00'
[2025-03-01T11:09:48.323+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-03-01T11:09:48.324+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '\n\t\t\tcd /opt/***\n                        dvc pull /opt/***/data/transformed\n                        python /opt/***/executables/data_storage.py && \t\t\tgit add logs &&                        git commit -m "Updated stored data version" -a &&                         dvc push && git push\n                    ']
[2025-03-01T11:09:48.334+0000] {subprocess.py:86} INFO - Output:
[2025-03-01T11:09:51.226+0000] {subprocess.py:93} INFO - Everything is up to date.
[2025-03-01T11:09:53.011+0000] {subprocess.py:93} INFO - 2025-03-01 11:09:53,010 : INFO : Connecting to Database
[2025-03-01T11:09:53.044+0000] {subprocess.py:93} INFO - 2025-03-01 11:09:53,044 : INFO : Connecting to Database Successfull
[2025-03-01T11:09:53.045+0000] {subprocess.py:93} INFO - 2025-03-01 11:09:53,044 : INFO : Reading Initial DB Setup Script
[2025-03-01T11:09:53.045+0000] {subprocess.py:93} INFO - 2025-03-01 11:09:53,044 : INFO : Executing script to Database
[2025-03-01T11:09:53.047+0000] {subprocess.py:93} INFO - 2025-03-01 11:09:53,046 : INFO : Executed script successfully !!
[2025-03-01T11:09:53.047+0000] {subprocess.py:93} INFO - 2025-03-01 11:09:53,046 : INFO : Reading insert script template
[2025-03-01T11:09:53.047+0000] {subprocess.py:93} INFO - 2025-03-01 11:09:53,047 : INFO : Inserting data to database
[2025-03-01T11:09:59.764+0000] {subprocess.py:93} INFO - 2025-03-01 11:09:59,763 : INFO : Executed script successfully !!
[2025-03-01T11:09:59.764+0000] {subprocess.py:93} INFO - 2025-03-01 11:09:59,764 : INFO : Connection closed !!
